airbyte-integrations/connectors/source-gocardless/source_gocardless/spec.yaml

```diff
@@ -0,0 +1,49 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  "$schema": http://json-schema.org/draft-07/schema#
+  title: Gocardless Spec
+  type: object
+  required:
+  - access_token
+  - gocardless_environment
+  - gocardless_version
+  - start_date
+  properties:
+    access_token:
+      title: Access Token
+      type: string
+      pattern: "^(sandbox|live)_.+$"
+      description: Gocardless API TOKEN
+      airbyte_secret: true
+    gocardless_environment:
+      title: Gocardless API Environment
+      type: string
+      pattern: "^(live|sandbox)$"
+      description: |
+        Environment you are trying to connect to. Should be either `live` or
+        `sandbox`.
```

Comment:
Done! Though I must say I couldn't find `enum` in the connector spec reference. Maybe I'm blind, but it would be nice if the docs had full-text search :)

Reply:
Yep, there isn't anything there. The docs must be updated.